SnapSheet/Classes/SnapSheetViewController.swift

```diff
@@ -64,9 +66,18 @@ public class SnapSheetViewController: UIViewController {
 	}()
 	private lazy var snap = UISnapBehavior(item: sheet, snapTo: CGPoint(x: self.sheetLayoutGuide.layoutFrame.midX, y: self.sheetLayoutGuide.layoutFrame.maxY))
 
+	public override func loadView() {
+		self.view = TouchForwardingView(frame: UIScreen.main.bounds)
+		self.view.autoresizingMask = [.flexibleHeight, .flexibleWidth]
+	}
+
 	public override func viewDidLoad() {
 		super.viewDidLoad()
 
+		if let parentView = self.presentingViewController?.view {
+			(view as? TouchForwardingView)?.passthroughViews.append(parentView)
```

Comment:
Q: doesn't this cause a retain cycle?

Reply:
Went over the whole deal with instruments, should all be memory ✅ now